src/Compilers/CSharp/Portable/Syntax/RecordDeclarationSyntax.cs

```diff
@@ -13,5 +15,22 @@ public partial class RecordDeclarationSyntax
                 return BaseList?.Types.FirstOrDefault() as PrimaryConstructorBaseTypeSyntax;
             }
         }
+
+        public RecordDeclarationSyntax Update(SyntaxList<AttributeListSyntax> attributeLists, SyntaxTokenList modifiers, SyntaxToken keyword, SyntaxToken identifier,
+            TypeParameterListSyntax? typeParameterList, ParameterListSyntax? parameterList, BaseListSyntax? baseList, SyntaxList<TypeParameterConstraintClauseSyntax> constraintClauses,
+            SyntaxToken openBraceToken, SyntaxList<MemberDeclarationSyntax> members, SyntaxToken closeBraceToken, SyntaxToken semicolonToken)
```

Comment:
Should the method allow the caller to decide whether to keep RecordModifier or not?

Reply:
That's not how our update methods typically work. In this case, we don't have a choice anyways (the signature is something we can't change).

Reply:
@jcouv So that's an acceptable breaking change right? Curious why the Public API Analyzer didn't flag the now-removed API?